src/reporting/market.se

```diff
@@ -168,9 +166,9 @@ def disputeAllReporters():
     assertNoValue()
     self.migrateThroughAllForks()
     require(self.isInAllDisputePhase())
-    self.getReputationToken().trustedTransfer(msg.sender, self, ALL_REPORTERS_DISPUTE_BOND_AMOUNT)
-    self.allReportersDisputeBondHolder = msg.sender
-    self.allReportersPayoutDistributionHash = self.tentativeWinningPayoutDistributionHash
+    require(self.limitedReportersDisputeBondToken)
```

Comment:
It's redundant since this function is also requiring self.isInAllDisputePhase(), which is checking to see if self.limitedReportersDisputeBondToken is set.  I'll remove it.